src/test/java/seedu/sellsavvy/logic/commands/ordercommands/AddOrderCommandTest.java

```diff
@@ -14,10 +13,6 @@ public class AddOrderCommandTest {
     @Test
     public void execute_newOrder_success() { // not a legit test
```

Comment:
I know it is not done yet but this method name needs to be changed.

Reply:
test cases settled